### PR TITLE
Update svc and pod after installation on k8s

### PR DESCRIPTION
**Pull request by jsenon**
Update service and pod after fresh installation. Replace istio-ingress by istio-ingresgateway

### CODE REVIEW

File: content/docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -282,20 +282,20 @@ Install Istio's core components. Choose one of the four _**mutually exclusive**_
...
-    istio-statsd-prom-bridge   ClusterIP      30.0.0.111   <none>          9102/TCP,9125/UDP                                                     7h
-    istio-telemetry            ClusterIP      30.0.0.246   <none>          9091/TCP,15004/TCP,9093/TCP,42422/TCP                                 7h
-    prometheus                 ClusterIP      30.0.0.253   <none>          9090/TCP                                                              7h
+    NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP        PORT(S)                                                               AGE
+    istio-citadel              ClusterIP      100.66.252.72    <none>             8060/TCP,9093/TCP                                                     2d
+    istio-egressgateway        ClusterIP      100.67.131.45    <none>             80/TCP,443/TCP                                                        2d
+    istio-ingressgateway       LoadBalancer   100.67.88.207    ac75b5b826267...   80:31380/TCP,443:31390/TCP,31400:31400/TCP                            2d
```
Comment (gyliu513) [inline content/docs/setup/kubernetes/quick-start.md]: Why did your `EXTERNAL-IP` is `ac75b5b826267..`?

Also I think you only need to update line 285, but do not need to touch other content.

Comment (jsenon) [reply content/docs/setup/kubernetes/quick-start.md]: We are using LoadBalancer on AWS that why you see an FQDN instead of IP

**Comment by jsenon** [comment]
@gyliu513 Do you prefer only modification on line 285? `istio-mixer-create-cr-bvv97` is also missing from pods overview (even if it's in status completed)

**Comment by gyliu513** [comment]
> @gyliu513 Do you prefer only modification on line 285? istio-mixer-create-cr-bvv97 is also missing from pods overview (even if it's in status completed)

Yes, I prefer only modify on line 285. There is no need to add `istio-mixer-create-cr-bvv97` as it is just a pod created by job, and the `kubectl` with version 1.9 will not show `Completed` pods.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 3e7776855a4896cc63acc1e8f26d01a78f403d1a

https://app.netlify.com/sites/preliminary-istio/deploys/5b0fb980dd6a5423f57bbf71

**Comment by jsenon** [comment]
External link http://cyan4973.github.io/xxHash had time-out. @geeknoid How can I relaunch netlify deployment?

**Pull request merged**